dev-python/poetry/poetry-1.3.1.ebuild

```diff
@@ -0,0 +1,57 @@
+# Copyright 2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
```

Comment:
Does it have tests?

Reply:
It does have test, but still depends on the `cachy` library. I do have an ebuild for that, but wanted to avoid adding it because the package is broken (bad pyproject.toml) and abandoned. Please advise.

Reply:
Ah, that's tricky. I'd suggest the following:
1. Report the issue to Poetry upstream asking them to reconsider the cachy test dep
2. Attempt to skip tests relying on cachy, if possible (hopefully there's some not using cachy).
3. If you are unsuccessful at 2, add RESTRICT="test" with an explanation of what you just told me + a link to the Poetry bug from Step 1